x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table_groups.tsx

```diff
@@ -68,6 +70,8 @@ export const SpikeAnalysisGroupsTable: FC<SpikeAnalysisTableProps> = ({
   dataView,
   timeRangeMs,
   searchQuery,
+  barColorOverride,
```

Comment:
Looks like these props aren't getting passed to the `SpikeAnalysisTable` in the expanded row (inside `toggleDetails`) for the grouped view:

<img width="1094" alt="image" src="https://github.com/elastic/kibana/assets/7405507/b27c6eae-2bfe-401e-aa75-54ca947dc50f">

Reply:
Great catch, fixed in 9d3cabd.